oak/common/app_config.h

```diff
@@ -27,6 +27,12 @@ namespace oak {
 // name and contents, accessible via gRPC.
 std::unique_ptr<ApplicationConfiguration> DefaultConfig(const std::string& module_bytes);
 
+// Reads a serialized application configuration from `file`.
+std::unique_ptr<ApplicationConfiguration> ReadConfigFromFile(const std::string& file);
```

Comment:
I assume `file` here and below is a file name, perhaps call them `filename`? In line with terminology from https://en.cppreference.com/w/cpp/io/basic_ofstream/basic_ofstream